### DISCUSSION TITLE
Please add cancel button for "Restore Default Shortcuts"

### DISCUSSION THREAD

**Discussion by sh-cho** [rxhanson/Rectangle#588] [Ideas]
![image](https://user-images.githubusercontent.com/11611397/135867980-fbe56df9-1a17-47d5-93e6-6f9632595aba.png)

I accidentally clicked "Restore Default Shortcuts" and there was no way out 😞  so I lost some settings.

Please add some cancel button

- version: v0.48 (53)

**Comment by rxhanson** [comment]
Thanks! For some reason I didn't think about that scenario. I just checked in a change for it.
